frontend/src/lib/components/HelpButton/HelpButton.scss

```diff
@@ -6,10 +6,6 @@
     font-size: 1.5rem;
     color: var(--primary-alt);
 
-    &:not(.custom-component) {
-        margin-left: 1.25rem !important;
-    }
-
```

Comment:
I looked around for use cases of this and I couldn't find one where this was necessary

Reply:
+1 for getting rid of the name "custom-component" anyway :) so fine to let this explode somewhere and fix forward if necessary